components/slack/sources/new-star-added/new-star-added.mjs

```diff
@@ -0,0 +1,24 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "slack-new-star-added",
+  name: "New Saved Message",
```

Comment:
This should be New Star Added, and sources that use webhooks need the word Instant in the name.
```suggestion
  name: "New Star Added (Instant)",
```